src/components/ReportActionItem/ReportPreview.js

```diff
@@ -260,7 +260,10 @@ function ReportPreview(props) {
     }, [isPaidGroupPolicy, isCurrentUserManager, isDraftExpenseReport, isApproved, iouSettled]);
     const shouldShowSettlementButton = shouldShowPayButton || shouldShowApproveButton;
     return (
-        <OfflineWithFeedback pendingAction={lodashGet(props, 'iouReport.pendingFields.preview')}>
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props, 'iouReport.pendingFields.preview')}
+            shouldDisableOpacity={props.action.pendingAction || props.action.isOptimisticAction}
+        >
```

Comment:
Just a quick question, of course, if you don't mind.

Similarly to the `shouldShowSettlementButton`, would it be cleaner if  `props.action.pendingAction || props.action.isOptimisticAction` had its variable such as `shouldDisableOpacity` or `hasPendingAction`?

For Example:

```
const shouldDisableOpacity = props.action.pendingAction || props.action.isOptimisticAction

 <OfflineWithFeedback
            pendingAction={lodashGet(props, 'iouReport.pendingFields.preview')}
            shouldDisableOpacity={shouldDisableOpacity}
```

Thanks

Reply:
Both are fine. We can leave the code as is.